dist/routable-component.js

```diff
@@ -381,37 +381,37 @@ var routableComponent =
 	var RoutableComponentRoutes = function () {
 	  /**
 	   * コンストラクタ
-	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routing_classes ルート定義クラス(の配列)
+	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routingClasses ルート定義クラス(の配列)
 	   */
 	
 	  function RoutableComponentRoutes() {
-	    var routing_classes = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+	    var routingClasses = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
 	    (0, _classCallCheck3.default)(this, RoutableComponentRoutes);
 	
 	    this._routes = [];
-	    this.include_route(routing_classes);
+	    this.includeRoute(routingClasses);
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->